<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainApp</class>
 <widget class="QDialog" name="MainApp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_14">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>301</width>
         <height>191</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="usernum">
          <property name="placeholderText">
           <string>Numéro de compte</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLineEdit" name="passwd">
            <property name="placeholderText">
             <string>Mot de passe</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="error_msg">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="connection">
            <property name="text">
             <string>Connexion</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="inscription">
            <property name="text">
             <string>Inscription</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QLabel" name="TitleLogin">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>40</y>
         <width>211</width>
         <height>31</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">* {
font-size: 20px;
}</string>
       </property>
       <property name="text">
        <string>Veuillez vous connecter</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <widget class="QWidget" name="verticalLayoutWidget_10">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>411</width>
         <height>381</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_23">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLineEdit" name="nameinput">
            <property name="placeholderText">
             <string>Nom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="firstnameinput">
            <property name="placeholderText">
             <string>Prénom</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="adress">
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Adresse</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="password">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="placeholderText">
             <string>Mot de passe</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="birth">
              <property name="text">
               <string>Date de naissance :</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioMan">
              <property name="text">
               <string>Homme</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioFemale">
              <property name="text">
               <string>Femme</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QRadioButton" name="radioNP">
              <property name="text">
               <string>Non précisé</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QDateEdit" name="birthDate">
            <property name="locale">
             <locale language="French" country="France"/>
            </property>
            <property name="timeSpec">
             <enum>Qt::UTC</enum>
            </property>
            <property name="date">
             <date>
              <year>2022</year>
              <month>12</month>
              <day>29</day>
             </date>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="cancel">
              <property name="text">
               <string>Retour</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="register_2">
              <property name="text">
               <string>S'inscrire</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_3">
      <widget class="QWidget" name="horizontalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>381</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Vos informations</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="toInfo">
            <property name="text">
             <string>Profil</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">* {
font-size: 15px;
}</string>
            </property>
            <property name="text">
             <string>Solde total :</string>
            </property>
           </widget>
          </item>
          <item alignment="Qt::AlignHCenter">
           <widget class="QLabel" name="sold_tofill">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>270</y>
         <width>383</width>
         <height>71</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="tolivret">
          <property name="text">
           <string>Créer un livret</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="virement">
          <property name="text">
           <string>Effectuer un virement</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="deconnexion">
          <property name="text">
           <string>Se déconnecter</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_12">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>70</y>
         <width>401</width>
         <height>52</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_17">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="IBAN">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>401</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_18">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_12">
          <property name="styleSheet">
           <string notr="true">* {
font-size:20px;
}</string>
          </property>
          <property name="text">
           <string>Bienvenue dans votre espace N27 ! </string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>40</y>
         <width>381</width>
         <height>31</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_15">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="nom_tofill">
          <property name="text">
           <string>Nom_tofill</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="prenom_tofill">
          <property name="text">
           <string>Prenom_tofill</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_4">
      <widget class="QWidget" name="verticalLayoutWidget_8">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>381</width>
         <height>61</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_21">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_2">
          <property name="styleSheet">
           <string notr="true">* {
font-size:20px;
}</string>
          </property>
          <property name="text">
           <string>Virement externe</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_9">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>70</y>
         <width>371</width>
         <height>311</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_22">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="label_5">
              <property name="styleSheet">
               <string notr="true">* {
font-size: 15px;
}</string>
              </property>
              <property name="text">
               <string>Votre solde :</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="sold_tofill_2">
              <property name="text">
               <string>sold_tofill</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit">
            <property name="placeholderText">
             <string>Montant du virement</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Compte émetteur :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="listAccount"/>
          </item>
          <item>
           <widget class="QLineEdit" name="RIB">
            <property name="placeholderText">
             <string>RIB destinataire</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QPushButton" name="toHome">
              <property name="text">
               <string>Retour</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="transfer">
              <property name="text">
               <string>Effectuer le virement</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_5">
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="styleSheet">
          <string notr="true">* {
font-size:23px;}</string>
         </property>
         <property name="text">
          <string>Votre virement a bien été effectué !</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_11">
         <item>
          <widget class="QPushButton" name="baktohome">
           <property name="text">
            <string>Retour</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_6">
      <layout class="QVBoxLayout" name="verticalLayout_10">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item alignment="Qt::AlignHCenter">
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">* {
font-size:20px;
}</string>
           </property>
           <property name="text">
            <string>Vos informations</string>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_13">
           <item alignment="Qt::AlignHCenter">
            <widget class="QLabel" name="idUserInfo">
             <property name="text">
              <string>TextLabel</string>
             </property>
             <property name="textInteractionFlags">
              <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByKeyboard</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_12">
                 <item>
                  <widget class="QLabel" name="nameInfo">
                   <property name="text">
                    <string>Nom Prénom</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="adressInfo">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="birthInfo">
                   <property name="text">
                    <string>Né le :</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QLabel" name="genreInfo">
                   <property name="text">
                    <string>TextLabel</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_15">
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeType">
                  <enum>QSizePolicy::Fixed</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QComboBox" name="accountbox"/>
               </item>
               <item alignment="Qt::AlignHCenter">
                <widget class="QLabel" name="sumAcc">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="tohomefrominfo">
             <property name="text">
              <string>Retour</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_7">
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>10</y>
         <width>171</width>
         <height>41</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">* {
font-size:20px;
}</string>
       </property>
       <property name="text">
        <string>Création d'un livret</string>
       </property>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_11">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>381</width>
         <height>321</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_16">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Type de livret : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="interest">
          <item>
           <property name="text">
            <string>Livret A</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Livret Jeune</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>Montant disponible sur ce livret : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="montant">
          <property name="placeholderText">
           <string>Entrer un montant (afin de tester les transactions)</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="text">
             <string>Retour</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="createAccount">
            <property name="text">
             <string>Créer</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_8">
      <widget class="QWidget" name="verticalLayoutWidget_3">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>381</width>
         <height>51</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_11">
          <property name="styleSheet">
           <string notr="true">* {
font-size:20px;
}</string>
          </property>
          <property name="text">
           <string>Sélectionnez un type de virement</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>60</y>
         <width>381</width>
         <height>321</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_13">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QPushButton" name="virIntern">
            <property name="text">
             <string>Virement Interne</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="virExtern">
            <property name="text">
             <string>Virement externe</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="page_9">
      <widget class="QWidget" name="verticalLayoutWidget_6">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>381</width>
         <height>80</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_19">
        <item alignment="Qt::AlignHCenter">
         <widget class="QLabel" name="label_13">
          <property name="styleSheet">
           <string notr="true">* {
font-size:20px;
}</string>
          </property>
          <property name="text">
           <string>Virement Interne</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>381</width>
         <height>241</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_20">
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>Compte émetteur :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="compteemet"/>
        </item>
        <item>
         <widget class="QLabel" name="soldeCompteemetteur">
          <property name="text">
           <string>SoldeCompteEmet</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>Compte récepteur :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="compterecept"/>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Montant : </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_2"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="horizontalLayoutWidget_7">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>340</y>
         <width>381</width>
         <height>41</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_16">
        <item>
         <widget class="QPushButton" name="tovir">
          <property name="text">
           <string>Retour</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tovirementemis">
          <property name="text">
           <string>Valider</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
